apps/site/pages/docs/building-sections/overrides/overrides-overview.mdx

```diff
@@ -0,0 +1,41 @@
+
+import { Callout } from 'nextra-theme-docs'
+
+<header>
+# Overrides
+</header>
+
+Overrides allow you to replace native components within sections with custom ones. This maintains the core functionalities of native components while allowing you to adjust their appearance and behavior to suit your store's needs.
+
+All component customizations happen within sections. You can create multiple overrides for native sections, resulting in a new React component representing the overridden section.
+
+<Callout type="info" emoji="ℹ️">
+  Sections are native components that group other components to create specifics part of your store page, such as the [Hero](https://www.faststore.dev/components/organisms/hero) which includes the following components: `Hero`, `HeroImage`, and `HeroHeader`,
+  simplifying the process of building cohesive layouts.
+</Callout> 
+
+## Override options
+
+There are three options to customize your component through overrides:
+
+- **Overriding component props:** Change a component's behavior by [passing additional props](https://www.faststore.dev/docs/building-sections/overrides/override-component-props) to it in your store code.
+ 
+- **Overriding a section:** [Build a new section](https://www.faststore.dev/docs/building-sections/overrides/override-component) based on its native equivalent to achieve the desired functionality.
+
+- **Overriding section styles:** Customize the look and feel of a section without changing its functionality through its [styles](https://www.faststore.dev/docs/building-sections/overrides/override-section-styles).
+
+## Available sections to override
+
+Check the available sections and their components in the [List of native sections](https://www.faststore.devhttp/docs/building-sections/overrides/list-of-native-sections) guide. 
+Also, check out the sections equivalent [Headless CMS schemas](https://developers.vtex.com/docs/guides/faststore/headless-cms-3-adding-content-types-and-sections#step-3-adding-sections-to-the-headless-cms). 
+
+Once you define which section you wish to modify, you can choose to override only its props or the whole section structure.
+
+### `__experimental` components
+
+Some sections may contain components labeled with `__experimental`. These components are undergoing migration to [FastStore UI](https://developers.vtex.com/docs/guides/faststore/components-index) and may present limitations. 
+While they are not yet fully migrated, `__experimental` components are available for use, but not for overriding options.
+
+## Headless CMS customization and overrides
```

Comment:
Actually we can place this information before the `experimental` section, since is more important.